openeo/local/connection.py

```diff
@@ -0,0 +1,316 @@
+import datetime
+import xarray as xr
+import numpy as np
+import rioxarray
+from glob import glob
```

Comment:
tip: if you are already using `pathlib`, I would just use `pathlib.Path.glob` instead of the old style `glob`

Reply:
Thanks, it's on my list!